docs/namespaces.md

```diff
@@ -19,21 +19,21 @@ a namespace binding section in the [configuration](/config) file for this.
 Setting this ensures that 2 instances of Chronicle will refer to the same
 namespace as 'default'.
 
-## Built-In Namespaces
+## Built-In namespaces
 
 ### default
 
 The default namespace is the namespace that is used when you do not specify one
-in a GraphQL mutation or query. It can be bound to any UUID, and should be set
+in a graphql mutation or query. It can be bound to any UUID, and should be set
```

Comment:
Seem to be many decapitalizations in this file, a rebase slip?

Reply:
Yes most likely